app/tests/spec/views/sign_up.js

```diff
@@ -528,10 +528,6 @@ define(function (require, exports, module) {
         view.showValidationErrors();
 
         assert.isTrue(view.showValidationError.called);
```

Comment:
> Do we have any functional tests for email validation?

If not, one or two on the signup page would probably be helpful. At least if we test the signup page, we can ensure invalid email addresses are not making their way to the DB.